lib/rexml/document.rb

```diff
@@ -431,14 +433,19 @@ def Document::entity_expansion_text_limit
     end
 
     attr_reader :entity_expansion_count
+    attr_accessor :entity_expansion_text_limit
```

Comment:
Do we need the reader for this?

Reply:
Yes
In the following we use reader.

- https://github.com/naitoh/rexml/blob/8554f73a8297838d994ee727199db18b4cfe4d85/lib/rexml/text.rb#L274
  > @unnormalized = Text::unnormalize(@value, parent, entity_expansion_text_limit: document.entity_expansion_text_limit)
- https://github.com/naitoh/rexml/blob/8554f73a8297838d994ee727199db18b4cfe4d85/lib/rexml/attribute.rb#L153
  > @unnormalized = Text::unnormalize(@normalized, doctype, entity_expansion_text_limit: @element.document.entity_expansion_text_limit)
- https://github.com/naitoh/rexml/blob/8554f73a8297838d994ee727199db18b4cfe4d85/lib/rexml/entity.rb#L80
  > @unnormalized = Text::unnormalize(@value, parent, entity_expansion_text_limit: document.entity_expansion_text_limit)